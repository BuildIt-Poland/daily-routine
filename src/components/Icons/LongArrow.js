import React from 'react';

function LongArrow() {
  return (
    <svg xmlns="http://www.w3.org/2000/svg" width="28" height="17" viewBox="0 0 28 17">
      <path fill="#fff" d="M19.5.015l-2.121 2.121L22.243 7H0v3h22.243l-4.864 4.864 2.121 2.121L27.985 8.5z" />
    </svg>
  );
}

export default LongArrow;
