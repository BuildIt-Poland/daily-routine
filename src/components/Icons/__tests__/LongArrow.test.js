import React from 'react';
import { create } from 'react-test-renderer';

import LongArrow from '../LongArrow';

describe('COMPONENT - Icons LongArrow', () => {
  it('renders correctly', () => {
    const component = create(<LongArrow />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
