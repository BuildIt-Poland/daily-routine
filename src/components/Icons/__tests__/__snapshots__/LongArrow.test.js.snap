// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Icons LongArrow renders correctly 1`] = `
<svg
  height="17"
  viewBox="0 0 28 17"
  width="28"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M19.5.015l-2.121 2.121L22.243 7H0v3h22.243l-4.864 4.864 2.121 2.121L27.985 8.5z"
    fill="#fff"
  />
</svg>
`;
